src/components/StateBrowser.vue

```diff
@@ -106,9 +106,14 @@ export default {
                 this.popup_networkid = null;
                 this.popup_top = 0;
             } else {
+                let h = 0;
+                let sb = document.getElementsByClassName('kiwi-statebrowser')[0];
+                if (sb.style && sb.style.top) {
```

Comment:
What if the `top` style is not on this element but its parent instead? Better to use `this.$el.getBoundingClientRect()` to get the `top` property instead (+ it will always be a number).

Reply:
Also, please add a comment to explain why it's applying this logic to the DOM. ie. `Some custom themes may push the statebrowser down, so apply any top to the popup also`.